dashboard/app/controllers/api/v1/pd/workshops_controller.rb

```diff
@@ -15,7 +15,18 @@ def index
       @workshops = @workshops.organized_by(current_user)
     end
 
-    render json: @workshops, each_serializer: Api::V1::Pd::WorkshopSerializer
+    if params[:include_enrollments]
+      enrollments = Pd::Enrollment.where(email: current_user.email)
+      return_val = []
+
+      @workshops.each do |workshop|
+        return_val << Api::V1::Pd::WorkshopSerializer.new(workshop, scope: enrollments.find_by(pd_workshop_id: workshop.id)).attributes
```

Comment:
Also since it's possible users may change their emails later, it's probably best if we match by user_id or email.

Something like:
```ruby
enrollment = workshop.enrollments.where('email = ? OR user_id = ?', current_user.email, current_user.id).first
```